### DISCUSSION TITLE
Latest 32.5.0 branch not compiling?

### DISCUSSION THREAD

**Discussion by tirankas** [OE4T/tegra-demo-distro#163] [General]
Hi Guys,
Updating to latest 32.5.0 dunfell branch I get this error during compile:

Applying patch 0010-tegra-mender-auto-configured-modified.patch
patching file configs/p2371-0000_defconfig
Hunk #1 FAILED at 1.
Hunk #2 FAILED at 45.
2 out of 2 hunks FAILED -- rejects in file configs/p2371-0000_defconfig
patching file configs/p2371-2180_defconfig
Hunk #2 FAILED at 51.
1 out of 2 hunks FAILED -- rejects in file configs/p2371-2180_defconfig
patching file configs/p2571_defconfig
Hunk #1 FAILED at 1.
Hunk #2 FAILED at 45.
2 out of 2 hunks FAILED -- rejects in file configs/p2571_defconfig
patching file configs/p2771-0000-000_defconfig
Hunk #2 FAILED at 36.
1 out of 2 hunks FAILED -- rejects in file configs/p2771-0000-000_defconfig
patching file configs/p2771-0000-500_defconfig
Hunk #2 FAILED at 36.
1 out of 2 hunks FAILED -- rejects in file configs/p2771-0000-500_defconfig
patching file configs/p3450-0000_defconfig
Hunk #1 FAILED at 54.
1 out of 1 hunk FAILED -- rejects in file configs/p3450-0000_defconfig
patching file configs/p3450-0002_defconfig
Hunk #1 FAILED at 50.
1 out of 1 hunk FAILED -- rejects in file configs/p3450-0002_defconfig

Is this known?

Thanks

**Comment by ichergui** [comment]
Hello @tirankas 
I didn't try to build `u-boot-tegra` for a while. which machine you are using ? 
I will fix it.
I will keep you updated.

**Reply by tirankas** [reply]
Thanks @ichergui 
I tried nano, emmc based device build

**Reply by ichergui** [reply]
As @madisongh is saying please make sure that everything is updated. Building `dunfell-l4t-r32.5.0` this weekend was successful

**Comment by madisongh** [comment]
Make sure you have updated your `meta-tegra` and `meta-tegra-community` layers to `dunfell-l4t-r32.5.0` latest as well.  The Mender patches are kept in meta-tegra-community now, and it sounds like your layers aren't quite in sync.

**Comment by tirankas** [comment]
Thanks Matt, was not aware of this

**Reply by tirankas** [reply]
Hi Matt,
Updating these layers, I see a dependency I have on tensorrt-container-csv is not provided in the latest dunfell 32.5.0 branch?
Is this expected?

**Reply by tirankas** [reply]
Also, I did update meta tegra and tegra community to the latest 32.5.0 and still see this issue. Am I missing something?
I mean I just cloned the tegra demo distro from clean and specifically for these submodules I did checkout the 32.5.0 branch directly

**Reply by tirankas** [reply]
I see the correct patches in the meta-tegra-community buy for some reason these are not picked up, instead it's picking from the meta-mender-community...
Shouldn't these patches be removed from the meta-mender-community or a layer priority changed to accommodate this?

**Reply by madisongh** [reply]
Did you update your bblayers.conf to add meta-tegra-community to it?

I'm not sure where the tensorrt-container-csv dependency is coming from, but the tensorrt recipes were refactored recently, so if you had any references to tensorrt in your own layers/recipes, they'll need updating.

**Reply by tirankas** [reply]
Yep, it's probably the issue, I have my own bblayer and it wasn't up to date, thanks Matt!
